tests/drivers/build_all/dts.fixup

```diff
@@ -0,0 +1,31 @@
+#if defined(CONFIG_HAS_DTS_I2C_DEVICE)
+
+#ifndef CONFIG_FXAS21002_NAME
+#define CONFIG_FXAS21002_NAME ""
+#define CONFIG_FXAS21002_I2C_ADDRESS 0
+#define CONFIG_FXAS21002_I2C_NAME ""
+#endif
+
+#ifndef CONFIG_FXOS8700_NAME
+#define CONFIG_FXOS8700_NAME ""
+#define CONFIG_FXOS8700_I2C_NAME ""
+#define CONFIG_FXOS8700_I2C_ADDRESS 0
```

Comment:
I think after this fix and #5698 we can add FXOS8700 back to this test, which will be nice.